glue/core/roi.py

```diff
@@ -865,7 +877,13 @@ def centroid(self):
 
         return np.dot(xs, dxy) * scl + x0, np.dot(ys, dxy) * scl + y0
 
-    def move_to(self, xdelta, ydelta):
+    def center(self):
+        return self.centroid()  # centroid is more robust than mean
```

Comment:
```suggestion
        # centroid is more robust than mean, but undefined for degenerate (1D) polygons
        if self.area() == 0:
            return self.mean()
        else:
            return self.centroid()
```
I think that special case needs to be taken care of (although it seems not to be tested :( ), just as done in https://github.com/glue-viz/glue/blob/d2adb6895514044d4b40fdf1f2c082ca92df6a4b/glue/core/roi.py#L887-L889 – `rotate_to` can then be simplified to just use
```python
        center = self.center() if center is None else center
```

Reply:
Good call. Done. Thanks!